src/Scouting.elm

```diff
@@ -1,18 +1,26 @@
-module Scouting exposing (Model, Msg, init, main, update, view)
+module Scouting exposing (Model, Msg, init, update, view)
 
 import Browser
-import Html exposing (input, text)
+import Element exposing (el, layout, padding)
+import Element.Input as Input exposing (labelAbove, labelHidden)
+import Html
 import Html.Attributes exposing (placeholder, style, value)
 import Html.Events exposing (onClick, onInput)
 import Maybe.Extra exposing (unwrap)
 
 
 inputWriter : String -> (String -> Msg) -> String -> Html.Html Msg
-inputWriter placeholderInput msg model =
-    Html.div [ style "margin-bottom" "1%" ]
-        [ input [ placeholder placeholderInput, onInput msg, value model ]
-            []
-        ]
+inputWriter placeholder msg model =
+    layout [] <|
```

Comment:
`layout` is meant to be used just inside `main` - where you have to convert `Element Msg` to `Html Msg`.